indico/modules/rb/client/js/common/bookings/BookingExportModal.jsx

```diff
@@ -0,0 +1,130 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2019 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import _ from 'lodash';
+import React from 'react';
+import {bindActionCreators} from 'redux';
+import {connect} from 'react-redux';
+import {Form as FinalForm} from 'react-final-form';
+import {Button, Form, Modal} from 'semantic-ui-react';
+import PropTypes from 'prop-types';
+
+import {Translate} from 'indico/react/i18n';
+import {FinalDatePeriod} from 'indico/react/components';
+import {FinalRadio} from 'indico/react/forms';
+import {actions as bookingActions} from '../bookings';
+import FinalRoomSelector from '../../components/RoomSelector';
+
+function validate({format, dates, rooms}) {
```

Comment:
Am I the only one who doesn't like the separate validation function (in favor of having validation functions specified on the Field itself)?